devel/libunwind/Portfile

```diff
@@ -49,14 +49,7 @@ destroot.dir    ${build.dir}
 if {${subport} eq "${name}-headers"} {
     revision 0
 
-    supported_archs noarch
-
-    build {}
-
-    destroot.target installhdrs
-    destroot.args \
-        PREFIX="${prefix}"
-    
+    # please keep this test *before* `supported_archs none`, which clears `configure.cxx_stdlib`
```

Comment:
none->noarch

Reply:
Thank you for noticing.
This change should now be made.